iModelCore/ecobjects/src/ECSchema.cpp

```diff
@@ -3446,12 +3446,29 @@ POP_DISABLE_DEPRECATION_WARNINGS
     return status;
     }
 
+/*---------------------------------------------------------------------------------**//**
+* Returns true if thisSchema indirectly references potentiallyReferencedSchema
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+static bool IndirectlyReferences(ECSchemaCR thisSchema, ECSchemaCR potentiallyReferencedSchema, ECN::SchemaMatchType matchType)

```

Comment:
This name is ambiguous. One might (even if it makes no sense) assume that it checks if a schema is indirectly referenced (and return false if it's directly referenced)
It's probably a good idea to refactor/rename the "DirectlyReferences" method and make it take a boolean parameter instead which sets the behavior (direct only vs all)